Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -33,7 +33,8 @@
 #   OFF by default. Enabled in test infrastructure only.
 # preview::
 #   Ready to be used in beta contexts like Safari Technology Preview.
-#   OFF by default. May be enabled automatically by clients.
+#   ON by default in Safari Technology Preview. OFF by default elsewhere.
```

Comment:
"OFF by default elsewhere" seems misleading too, as it's enabled in test infrastructure. This is called out explicitly for `testable` above, so this omission is non-trivial.

Reply:
Will fix.

Reply:
https://github.com/WebKit/WebKit/pull/29852